compiler/ballerina-backend-jvm/src/main/ballerina/src/compiler_backend_jvm/jvm_instruction_gen.bal

```diff
@@ -910,6 +911,16 @@ type InstructionGenerator object {
                                     io:sprintf("(L%s;L%s;)V", FUNCTION, BTYPE), false);
         }
 
+        // Set annotations if available.
+        self.mv.visitInsn(DUP);
+        string pkgClassName = pkgName == "." || pkgName == "" ? MODULE_INIT_CLASS_NAME :
+                                    lookupGlobalVarClassName(pkgName + ANNOTATION_MAP_NAME);
+        self.mv.visitFieldInsn(GETSTATIC, pkgClassName, ANNOTATION_MAP_NAME, io:sprintf("L%s;", MAP_VALUE));
+        self.mv.visitTypeInsn(CHECKCAST, MAP_VALUE);
```

Comment:
do we need this checkcast?

Reply:
Fixed with 430e1d039ef2e44f02f2e839c432a5636fd4224b, thanks.